src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/AnnotationUnitTest.java

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.variant.vcf.VCFHeaderLine;
+import org.broadinstitute.hellbender.engine.FeatureInput;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class AnnotationUnitTest extends BaseTest {
```

Comment:
Move these tests to `VariantAnnotatorEngineUnitTest` instead of creating a separate class?